<launch>

  <node pkg="mapviz" type="mapviz" name="mapviz"/>

  <node pkg="swri_transform_util" type="initialize_origin.py" name="initialize_origin">
    <param name="local_xy_frame" value="/map"/>
    <param name="local_xy_origin" value="auto"/>  <!-- setting "auto" here will set the origin to the first GPS fix that it recieves -->
    <rosparam param="local_xy_origins">
      [{ name: custom,
         latitude: 45.49853,
         longitude: -73.63124,
         altitude: 0.0,
         heading: 0.0}]
    </rosparam>
  </node>

  <node pkg="tf" type="static_transform_publisher" name="swri_transform" args="0 0 0 0 0 0 /map /origin 100"/>

  <node pkg="rover_nav" type="pathMgr.py" name="pathMgr"/>

</launch>
